tests/integration/cases/jenkinsci_plotplugin/policy.dl

```diff
@@ -1,12 +1,12 @@
-/* Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved. */
+/* Copyright (c) 2024 - 2025, Oracle and/or its affiliates. All rights reserved. */
 /* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
 
 #include "prelude.dl"
 
 Policy("test_policy", component_id, "") :-
-    check_passed(component_id, "mcn_build_script_1"),
-    check_passed(component_id, "mcn_build_service_1"),
     check_passed(component_id, "mcn_version_control_system_1"),
+    check_passed(component_id, "mcn_build_script_1"),
+    check_failed(component_id, "mcn_build_service_1"),
```

Comment:
I wonder why `mcn_build_service_1` is changed from PASSED to FAILED ?

Reply:
The previous `PASSED` result was a false positive.